lib/kafka_ex/server_0_p_8_p_2.ex

```diff
@@ -55,10 +55,15 @@ defmodule KafkaEx.Server0P8P2 do
     state = update_metadata(state)
     {:ok, _} = :timer.send_interval(state.metadata_update_interval, :update_metadata)
 
-    # only start the consumer group update cycle if we are using consumer groups
-    if consumer_group?(state) do
-      {:ok, _} = :timer.send_interval(state.consumer_group_update_interval, :update_consumer_metadata)
-    end
+    state =
```

Comment:
@bjhaid or @cdegroot do you know if this feature is supported on 0.8.2?  I don't think it is, but was hoping one of you would know off-hand.